wp-content/plugins/core/src/Templates/Single.php

```diff
@@ -102,4 +106,42 @@ protected function get_pagination_item( $post_id ): string {
 		return $link->render();
 	}
 
+	protected function get_related_posts_cards() {
```

Comment:
TBD on this and as we have discussed, it'll be part of a much larger convo and work. We'll also ultimately want BE to take a look and see what, if anything more, we'd like to do with related content as a thing as I could see this turning into a more globally available/reusable and robust thing 😃 

Nice job on getting in here and taking a first crack at it and also asking some great questions. Nice job in using the card component as well 🏅